pkg/v1/remote/transport/bearer_test.go

```diff
@@ -192,12 +192,12 @@ func TestBearerTransportTokenRefresh(t *testing.T) {
 	if res.StatusCode != http.StatusOK {
 		t.Errorf("client.Get final StatusCode got %v, want: %v", res.StatusCode, http.StatusOK)
 	}
-	if got, want := transport.bearer.RegistryToken, refreshedToken; got != want {
+	if got, want := transport.bearer[0].RegistryToken, refreshedToken; got != want {
```

Comment:
Could we have a test for a multi-keychain that has some good and some bad (and some expired and needing refreshing) tokens, like we have in base_test.go?

Reply:
fixed